Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -160,8 +189,42 @@ private Operation CreateGetOperation(OpenApiMetadataPathsResource openApiMetadat
                 {
                     _contentTypeStrategy.GetOperationContentType(openApiMetadataResource, ContentTypeUsage.Readable)
                 },
-                parameters = CreateGetByExampleParameters(openApiMetadataResource, isCompositeContext),
-                responses = CreateReadResponses(openApiMetadataResource, true)
+                parameters = CreateGetByExampleParameters(openApiMetadataResource, isCompositeContext, true),
+                responses = responses
+            };
+
+            return operation;
+        }
+
+        private Operation CreateGetPartitionsOperation(OpenApiMetadataPathsResource openApiMetadataResource)
+        {
+            var parameters = new Parameter[]
+            {
+                new() {@ref = OpenApiMetadataDocumentHelper.GetParameterReference("numberOfPartitions")}
+            }
+            .Concat(CreateGetByExampleParameters(openApiMetadataResource, false, false));
+
+            var responses = OpenApiMetadataDocumentHelper.GetBaseResponses();
+            responses.Add("200", new Response {@ref = OpenApiMetadataDocumentHelper.GetResponseReference("Partitions")});
+
+            var operation = new Operation
+            {
+                tags = new List<string>
+                {
+                    OpenApiMetadataDocumentHelper.GetResourcePluralName(openApiMetadataResource.Resource)
+                        .ToCamelCase()
+                },
+                summary = "Retrieves a set of page tokens to be used for key set paging in the \"Get All\" endpoint.",
```

Comment:
This text appears here:

![image](https://github.com/user-attachments/assets/2fa203e4-d4a0-47e3-8925-cee31ddf3c0d)

Reply:
Suggestion for this text:

- Retrieves a set of page tokens to be used for efficient client-side parallel processing.

Reply:
Changed.